crates/bevy_math/src/cubic_splines.rs

```diff
@@ -321,6 +596,20 @@ impl<P: Point> CubicSegment<P> {
         let [_, _, c, d] = self.coeff;
         c * 2.0 + d * 6.0 * t
     }
+
+    #[inline]
+    fn coefficients(p: [P; 4], char_matrix: [[f32; 4]; 4]) -> Self {
```

Comment:
Doc strings please.

Reply:
Even though this function is private, I do agree that it would benefit from some documentation. Added it.